test/e2e/lib/gutenberg/blocks/blog-posts-block-component.js

```diff
@@ -6,6 +11,9 @@ import GutenbergBlockComponent from './gutenberg-block-component';
 class BlogPostsBlockComponent extends GutenbergBlockComponent {
 	static blockTitle = 'Blog Posts';
 	static blockName = 'a8c/blog-posts';
+	static blockFrontendSelector = By.css(
+		'.entry-content .wp-block-newspack-blocks-homepage-articles'
```

Comment:
These are pretty regular, can't we derive the name from the `blockName` (like Gutenberg calculates it) instead of defining them?

Looking at how this is currently structured, would it be more inline to add a method like `selectFrontendBlock` on the superclass?

https://github.com/Automattic/wp-calypso/blob/b81c4fba3882f13865521a4e0a34c008307fd12c/test/e2e/lib/gutenberg/blocks/gutenberg-block-component.js#L12-L33

Reply:
Agreed 👍🏻 I'll deal with that soon, perhaps in another PR (added it to our tracking issue, [here](https://github.com/Automattic/wp-calypso/issues/45209)).